### DISCUSSION TITLE
I accidently added a large amount of text into a title. I cannot delete it as no space is available on screen to go to bottom and delete it.

### DISCUSSION THREAD

**Discussion by apetivist** [alainm23/planify#833] [Q&A]
I apologize as this may or may not be some level of design flaw. I know it was my mistake. Is there some way to fix this? Must I remove Planner and reinstall and if so how can I assure that same mistake title doesn't show again? I am a noob to Linux.

Using Zorin OS 16 Pro.

**Comment by alainm23** [comment]
¿Could you send a screenshot please?

**Reply by apetivist** [reply]
I will try but in doing so must use Obfuscate to not release details in the title.

**Reply by apetivist** [reply]
![Screenshot from 2021-12-15 14-19-30](https://user-images.githubusercontent.com/61020619/146260031-07b2908f-b6d1-4f5e-9aea-cd6949cf191c.png)

**Reply by apetivist** [reply]
Thank you for your help. I love Planner!

**Comment by alainm23** [comment]
- Try editing the project and change the title to a shorter one. Right click on the project > Edit

**Reply by apetivist** [reply]
Done! Thank you. I don't know why I got stuck in the "Delete' mode of thinking. 
:)
